test(onboarding): fix signup with seed phrase tests

jrainville opened the pull request:
Fixes #6744

The seed phrase tests were pretty messed up. I don't even know if they ever worked.

Anyway, they are upgraded and cleaned up.

StatusQ PR: https://github.com/status-im/StatusQ/pull/820
Branched on top of: https://github.com/status-im/status-desktop/pull/6735

jrainville commented:
@endulab you are the tester for this PR (test the tests lol)

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 9e5be22b | [#1](https://ci.status.im/job/status-desktop/job/branches/job/macos/job/PR-6766/1/) | 2022-08-01 22:05:06 | ~8 min | `macos` | [:package:`dmg`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Desktop-220801-215640-9e5be2-pr6766.dmg) |

alaibe commented on test/ui-test/src/screens/StatusWelcomeScreen.py:
```diff
@@ -59,17 +84,46 @@ def agree_terms_conditions_and_navigate_to_import_seed_phrase(self):
+        else:
+            test.fail("Wrong amount of seed words", len(words))
+
+        type(SeedPhraseComponents.SEEDS_WORDS_TEXTFIELD_1.value, words[0])
```
maybe add a function, we do same thing in the wallet test?

alaibe reviewed (approved):
Yeah they never worked for me either

jrainville commented:
@alaibe done. Can you give it a second look?

endulab commented:
@jrainville I ran tests and there is a fail:

![image](https://user-images.githubusercontent.com/61889657/183046365-53617c5d-ab1a-409a-9ff3-6e6d65533e3a.png)

endulab reviewed (changes requested):
- [x] fix failing Squish test

jrainville commented:
@endulab thanks  for the test. Turns out I wasn't clear in my description, but I had fixed the signUp test and not the login test.

Anyway, I fixed the login test as well, so both should work now.

endulab commented:
![image](https://user-images.githubusercontent.com/61889657/183116726-dfbf7221-e80a-4175-9bb8-305813128694.png)
